<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuboBalanceManage</class>
 <widget class="QDialog" name="HuboBalanceManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="goBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>261</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item row="3" column="1">
     <widget class="QTextEdit" name="textEdit_4">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>k linear</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>k anglular</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>w track</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="textEdit_5">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>kd</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit_2">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>kp</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit_3">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>frameRate</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="textEdit_6">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
